main.py

```diff
@@ -40,12 +40,13 @@ def run() -> None:
     logger.info("Starting Living Documentation generation.")
 
     ActionInputs().validate_repositories_configuration()
+    output_formats: list[str] = ActionInputs.get_output_formats()
```

Comment:
Why it has to be used here.
Is possible to ask for this value inside of generator when really needed?

Reply:
You are right, this is not needed run in the main script. I did extend the action input validation method, and move the getter into the correct place. See commit: TBD.